### DISCUSSION TITLE
RFC: The standard of moving package from core-testing/extra-testing to core/extra

### DISCUSSION THREAD

**Discussion by wszqkzqk** [lcpu-club/loongarch-packages#147] [Ideas]
Our scripts now can only move **ALL** packages of one repo to another, so the process of moving testing packages need to be discussed.

## Which packages need to be uploaded to testing first?

* **ALL** `core` packages need to be passed to `core-testing`
* `extra` packages are **optional** to be uploaded to testing
  * Recommend to follow the same choice of Arch Linux official

**Comment by setarcos** [comment]
It'll be pointless if there is no user of the testing repos.  What we need here is a testing procedure.  What makes a package stable enough to move to the stable repos.   We need actual tester to make the testing really works.

**Reply by wszqkzqk** [reply]
But we already have had testing repos and used it in some cases.

**Reply by setarcos** [reply]
The only criteria I can think of is "package has been in testing repo for 2 weeks without complain."
